AdblockPlusSafari/Base.lproj/Localizable.strings

```diff
@@ -0,0 +1,20 @@
+/* 
```

Comment:
How come there are only four strings in this file? Seems to me we're either missing some strings here, or we don't need those we have here.

If we do need the file, please use the standard file header.